ros/api/common/utils.py

```diff
@@ -47,8 +48,8 @@ def check_for_rating(data):
     def check_for_user():
         ident = identity(request)['identity']
         user = user_data_from_identity(ident)
-        username = user['username'] if 'username' in user else None
-
+        service_account_user = service_account_from_identity(ident)
```

Comment:
```suggestion
        service_account_info = service_account_from_identity(ident)
```
To be more specific as it returns `service_account` object not `user` object.